docs/procesos/P14-proceso-de-definicion-del-plan-inicial.md

```diff
@@ -7,21 +7,98 @@
-| 2. Estimación | <ul><li>Proponer una solución. </li><li>Definir el WBS del plan [(guía)](../guias/G07-guia-wbs) </li><li>Definir la estimación de cada work item  </li><li>Identificar la disponibilidad de los miembros del plan.</li><li>Estimar el calendario del plan</li><li>Definir los hitos del plan .</li> <li>Documentar la estimación [(Usar la plantilla de documentación de un plan)](../plantillas/PL08-plantilla-para-documentacion-de-un-plan)</li> </ul> | <ul><li>PP 1.1</li><li>PP 1.2</li><li>PP 2.1</li><li>PP 2.7</li></ul> |
-| 3. Solución | <ul><li>Obtener compromiso de todos los interesados.[(Documentar)](../plantillas/PL08-plantilla-para-documentacion-de-un-plan)</li> <li>Verificar la solución.</li> </ul> | <ul><li>PP 2.7</li></ul> |
+
+<table>
+  <thead>
+    <th>Fase</th>
+    <th>Actividades</th>
+    <th>Área(s) involucradas</th>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>1. Identificación</td>
+      <td>
+        <ul align="left">
+          <li>Definir un ciclo de vida para el plan. </li>
+            <ul align="left">
+              <li>Tipos de ciclos de vida a elegir:</li>
+              <ul align="left">
+                <li><a href="https://docs.google.com/presentation/d/1zTMVnqxYRWe_Fh0TFrjlKbB42MXc9lM4ftcBvG49oJ8/edit#slide=id.g2b2b7a0211_0_6">Ciclo de vida de DAD clásico.</a></li>
+                <li><a href="https://docs.google.com/presentation/d/1zTMVnqxYRWe_Fh0TFrjlKbB42MXc9lM4ftcBvG49oJ8/edit#slide=id.g2b2b7a0211_0_19">Ciclo de vida de DAD avanzado.</a></li>
+                <li><a href="https://docs.google.com/presentation/d/1zTMVnqxYRWe_Fh0TFrjlKbB42MXc9lM4ftcBvG49oJ8/edit#slide=id.g2b2b7a0211_0_115">Ciclo de vida de entrega continua ágil.</a></li>
+                <li><a href="https://docs.google.com/presentation/d/1zTMVnqxYRWe_Fh0TFrjlKbB42MXc9lM4ftcBvG49oJ8/edit#slide=id.g2b2b7a0211_0_122">Ciclo de vida de entrega continua lean.</a></li>
+                <li><a href="https://docs.google.com/presentation/d/1zTMVnqxYRWe_Fh0TFrjlKbB42MXc9lM4ftcBvG49oJ8/edit#slide=id.g2b2b7a0211_0_129">Ciclo de vida exploratorio.</a></li>
+              </ul>
+            </ul>
+          <li>Reunir las habilidades<a href="https://docs.google.com/spreadsheets/d/1MZ-7gZ1iUZPbCIa3G2UtK8B-_w1ipbHrO0hJMeRTozQ/edit#gid=1760954040"> (Utilizar el documento de Trials of the temple).</a> y recursos <a href="https://docs.google.com/spreadsheets/d/1frtMUtfqJzUiE9ej_qi-HwhZZtvbSbPsc-TMsYf10Bk/edit#gid=0">(Utilizar el repositorio de recursos departamentales)</a> que se tienen en el momento.<a></a></li>
+          <li>Analizar las habilidades que les faltan a las personas que son necesarias para el plan <a href="https://docs.google.com/spreadsheets/d/1MZ-7gZ1iUZPbCIa3G2UtK8B-_w1ipbHrO0hJMeRTozQ/edit#gid=1760954040"> (Utilizar el documento de Trials of the temple).</a></li>
```

Comment:
Lo mismo aquí --> "para el plan, utilizando el documento <a>Trials of the temple</a>.